### DISCUSSION TITLE
${__from:date} doesn't work in prometheus labels?

### DISCUSSION THREAD

**Discussion by tedpearson** [grafana/grafana#42904] [General]
Grafana version: 8.3.1
Running on linux armhf (raspberry pi)

I'm trying to query a metric in a prometheus datasource where I have a label with a formatted date I'd like to query based on a $__from or $__to variable. But the query inspector tells me that a format like `"${__to:date:YYYY-MM-DD}"` is sending the raw string as the query and not interpolating the variable. 

Example query not working as expected: `metric{time_label="${__to:date:YYYY-MM-DD}"}`

Is this expected behavior, or should to/from "special formatting syntax" be working in label queries?

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
